tests/post_training/model_scope.json

```diff
@@ -0,0 +1,235 @@
+{
+    "vgg11": {
+        "model_name": "vgg11",
+        "quantization_params": {},
+        "metrics": {
+            "FP32 top 1": 0.61246
+        }
+    },
+    "resnet18": {
+        "model_name": "resnet18",
+        "quantization_params": {},
+        "metrics": {
+            "FP32 top 1": 0.62734
+        }
+    },
+    "mobilenetv2_050": {
+        "model_name": "mobilenetv2_050",
+        "quantization_params": {
+            "preset": "MIXED"
+        },
+        "metrics": {
+            "FP32 top 1": 0.59498
+        }
+    },
+    "mobilenetv2_050_BC": {
+        "model_name": "mobilenetv2_050",
+        "quantization_params": {
+            "preset": "MIXED",
+            "fast_bias_correction": false
+        },
+        "metrics": {
+            "FP32 top 1": 0.59498
+        }
+    },
+    "densenet121": {
+        "skipped": "temporary excluded due to memory leaks",
+        "model_name": "densenet121",
+        "quantization_params": {
+            "fast_bias_correction": false
+        },
+        "metrics": -1
```

Comment:
Since we removing `-1` from the resulting table I suggest doing this here too.

Reply:
remove it, `get_cached_metric` will return None if metric does not exists.